db/dbtypes/types.go

```diff
@@ -103,6 +111,44 @@ func (p TicketPoolStatus) String() string {
 	}
 }
 
+func (v VoteChoice) String() string {
+	switch v {
+	case Abstain:
+		return "abstain"
+	case Yes:
+		return "yes"
+	case No:
+		return "no"
+	default:
+		return "unknown"
+	}
+}
+
+// ToChoiceIndex converts the vote choice string to a
+// vote choice index.
+func ToChoiceIndex(choice string) (VoteChoice, error) {
+	switch choice {
+	case "abstain":
+		return Abstain, nil
+	case "yes":
+		return Yes, nil
+	case "no":
+		return No, nil
+	default:
+		return 0, fmt.Errorf(`Vote Choice "%s" is unknown`, choice)
+	}
+}
+
+// MileStone defines the various stages passed by vote on a given agenda.
+// Activated is the height at which the delay time before a yes vote activates.
```

Comment:
...at which the delay time begins before a vote activates.

Reply:
Noted